docs/glossary.md

```diff
@@ -74,187 +215,135 @@ license: "This software is licensed under the Apache License version 2."
+- A automation tool for building, testing, and deploying projects.
 
 ## Gradlew
 
-- A Gradle wrapper script, used for executing Gradle commands without installing Gradle separately.
-
-## Apache Gravitino
-
-- An open-source software platform originally created by Datastrato for high-performance, geo-distributed, and federated metadata lakes. Designed to manage metadata directly in different sources, types, and regions, providing unified metadata access for data and AI assets.
-
-## Apache Gravitino configuration file (gravitino.conf)
-
-- The configuration file for the Gravitino server, located in the `conf` directory. It follows the standard property file format and contains settings for the Gravitino server.
+- A Gradle wrapper script, used for executing Gradle commands.
 
 ## Hashes
 
-- Cryptographic hash values generated from the contents of a file, often used for integrity verification.
-
-## HDFS
-
-- **HDFS** (Hadoop Distributed File System) is an open-source, distributed file system and a key component of the Apache Hadoop ecosystem. It is designed to store and process large-scale datasets, providing high reliability, fault tolerance, and performance for distributed storage solutions.
+- Cryptographic hash values generated from some data.
+  A typical use case is to verify the integrity of a file.
 
 ## Headless
 
-- A system without a graphical user interface.
-
-## HTTP port
-
-- The port number on which a server listens for incoming connections.
-
-## Apache Iceberg Hive catalog
-
-- The **Iceberg Hive catalog** is a specialized metadata service designed for the Apache Iceberg table format, allowing external systems to interact with Iceberg metadata via a Hive metastore thrift client.
-
-## Apache Iceberg REST catalog
-
-- The **Iceberg REST Catalog** is a specialized metadata service designed for the Apache Iceberg table format, allowing external systems to interact with Iceberg metadata via a RESTful API.
-
-## Apache Iceberg JDBC catalog
-
-- The **Iceberg JDBC Catalog** is a specialized metadata service designed for the Apache Iceberg table format, allowing external systems to interact with Iceberg metadata using JDBC (Java Database Connectivity).
+- A system without a local console.
 
 ## Identity fields
 
-- Fields in tables that define the identity of the table, specifying how rows in the table are uniquely identified.
+- Fields in tables that define the identity of the records.
+  In the scope of a table, the identity fields are used as the unique identifier of a row.
 
 ## Integration tests
 
-- Tests designed to ensure the correctness and compatibility of software when integrated into a unified system.
-
-## IP address
-
-- Internet Protocol address, a numerical label assigned to each device participating in a computer network.
+- Tests for the correctness and compatibility of a software.
+  It is typically conducted when integrating a component into a larger system.
```

Comment:
I think this is better: "Tests that ensure software correctness and compatibility when integrating components into a larger system."